assets/src/dashboard/app/api/useUserApi.js

```diff
@@ -48,11 +48,39 @@ export default function useUserApi(dataAdapter, { currentUserApi }) {
     }
   }, [dataAdapter, currentUser, currentUserApi]);
 
+  const toggleWebStoriesMediaOptimization = useCallback(async () => {
+    setIsUpdating(true);
+    try {
+      setCurrentUser(
+        await dataAdapter.post(currentUserApi, {
+          data: {
+            meta: {
+              web_stories_media_optimization: !currentUser.meta
+                .web_stories_media_optimization,
```

Comment:
`!currentUser.meta?.web_stories_media_optimization,` instead if there's a chance that the .ewb_stories_media_optimization isn't present?

Reply:
It will new be null or not exist. We makes sure to add default value. See here 
https://github.com/google/web-stories-wp/blob/21d3117d9db658935c018cfdbc2ba2f1617d0649/includes/User_Preferences.php#L85